source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -485,9 +485,45 @@ def _set_specific_params(mod):
 
         # Add the interpreted parameters to the stats
         self._register_interpreted_parameters(args)
-        logging.getLogger(__name__).debug(
-            "We are about to ask journeys on regions : {}".format(possible_regions)
-        )
+
+        logger = logging.getLogger(__name__)
+
+        logger.debug("We are about to ask journeys on regions : {}".format(possible_regions))
+
+        # generate an id that :
+        #  - depends on the coverage and differents arguments of the request (from, to, datetime, etc.)
+        #  - does not depends on the order of the arguments in the request (only on their values)
+        #  - does not depend on the _override_scenario argument
+        #  - if  _override_scenario is present, its type is added at the end of the id in
+        #    order to identify identical requests made with the different scenarios
+        #  - we add the current_datetime of the request so as to differentiate
+        #    the same request made at distinct moments
+        def generate_request_id():
+            import flask, json, hashlib
+
+            path = str(flask.request.path)
+            args_for_id = dict(flask.request.args)
+            if "_override_scenario" in args_for_id:
+                scenario = str(args_for_id["_override_scenario"])
+                args_for_id["_override_scenario"] = ""
+            else:
+                scenario = ""
+
+            json_repr = json.dumps(args_for_id, sort_keys=True, ensure_ascii=True)
+
+            # we could use the json_repr as an id, but we hash it to have something smaller
+            m = hashlib.sha256()
+            m.update(json_repr)
+            json_hash = m.hexdigest()
+
+            result = "journeys_{}_{}_{}".format(json_hash, scenario, args['_current_datetime'])
+
+            logger.debug("Generating id {} for request {}".format(result, json_repr))
+
+            return result
+
+        request_id = generate_request_id()
+        args["request_id"] = request_id
```

Comment:
> OK, you want to be able to compare the response time between two scenarios right?
Exactly, since the changes in response time between new_default and distributed seems to vary with the request, I thought it may be useful to have the same request_id for the same request. 
Maybe an overkill though